docs/configuration.rst

```diff
@@ -26,6 +26,12 @@ Required services
 PAPERLESS_REDIS=<url>
     This is required for processing scheduled tasks such as email fetching, index
     optimization and for training the automatic document matcher.
+    
+    * If your Redis server needs login credentials ``PAPERLESS_REDIS=redis://<username>:<password>@<ip>:<port>``
```

Comment:
Nit: It doesn't need to be an IP, a hostname will also work.  Maybe something like `<host | ip>`?  Or just `<host>`?

Reply:
You're right, in the documentation of redis it's <host> and not <ip>.